src/main/resources/static/_shared.yml

```diff
@@ -255,8 +255,11 @@ components:
           type: string
           format: uuid
           example: 6abb5fa3-e93f-4445-887b-30d081688f44
+        personSummary:
+          $ref: '#/components/schemas/PersonSummary'
         personName:
           type: string
+          deprecated: true
```

Comment:
could we please add a 'description' property describing what should be used instead of this deprecated field (personSummary). If you search for other deprecate fields in this file it'll provide an example